src/js/components/reviewData/ReviewDataNarrative.jsx

```diff
@@ -26,7 +28,8 @@ export default class ReviewDataNarrative extends React.Component {
             currentFile: "A",
             fileNarrative: {},
             currentNarrative: "",
-            saveState: ""
+            saveState: "",
+            errorMessage: false
```

Comment:
Since `errorMessage` is later set to a string, it should initially default to the empty string to keep the type consistent

Reply:
Ah yeah that was leftover from testing around different implementation ideas. Fixing.